### PR TITLE
Server user schema

### PR DESCRIPTION
**Pull request by phavor**
Setup the userSchema and routes

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const userRoute = require('./routes/user')
+const error = require('./middleware/errors')
+
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: true }))
+app.use(logger('dev'))
+app.use(cors())
+
+// connect mongoose
+mongoose.connect(URI, { useNewUrlParser: true })
```
Comment (Youngestdev) [inline backend/server.js]:
```suggestion
mongoose.connect(URI, { useNewUrlParser: true, useCreateIndex: true } )
```

Comment (phavor) [reply backend/server.js]: Nodemon will be installed as a devDependencies

File: backend/controllers/user.js [hunk 2]
```diff
@@ -15,10 +15,12 @@ exports.signup = async (req, res) => {
     message: 'user already registered'
   })
 
+  console.log(req.body)
```
Comment (Youngestdev) [inline backend/controllers/user.js]: Remove this.

Comment (phavor) [reply backend/controllers/user.js]: Lolzzzz, how did I miss that.

**Pull request merged**